.circleci/config.yml

```diff
@@ -54,6 +54,9 @@ jobs:
             git config user.email "support@spry-group.com"
             git config user.name "Spry Bot"
             git commit -m $'chore: update content.json\n\n[skip ci]'
```

Comment:
We only need one commit here when everything succeeds.

Reply:
@dopry updated